src/plugins/doaction/doaction-doc-en.hbs

```diff
@@ -296,6 +297,8 @@
 <dd>Load a JSON file into a specified <a href="../jsonmanager/jsonmanager-doc-en.html">JSON manager</a></dd>
 <dt>withInput</dt>
 <dd>Specify the value of an input field available to the action</dd>
+<dt>openDialog</dt>
+<dd>Open the overlay</dd>
```

Comment:
Document here that the overlay makrup need to be next to the button with the do-action.